integration/pkg/api/search/nearbychargestation/mock_response.go

```diff
@@ -0,0 +1,193 @@
+package nearbychargestation
```

Comment:
use `mock_response_test.go` for test only files.

Reply:
I think change to `mock_response_test.go` need special handling for elements inside, by default, the definition inside `*_test` file is not exposed:
![image](https://user-images.githubusercontent.com/16873751/75810750-02885280-5d40-11ea-8386-47cb9a38ad2a.png)

This old link mentioned the *_test.go files are compiled into the package only when running go test for that package.
https://github.com/golang/go/issues/10184

Reply:
Yes, that's right. I may understand it incorrect. It's part of real function, just no implementation now. I'm ok with it, and when the functions implemented, I think we don't need any `mock_xxx` in codes except test, right?

Reply:
Yes.  For all mock related topic will be started with `Mock` prefix and will only be used in test.